tests/integration/models/gpflux/test_bayesian_optimization.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 The Trieste Contributors
```

Comment:
I think this should go to `tests/integration/test_bayesian_optimization.py` test file rather than specific models folder